lib/asciidoctor/converter/semantic_html5.rb

```diff
@@ -80,6 +224,63 @@ def convert_inline_anchor node
     end
   end
 
+  def convert_inline_quoted node
+    attributes = common_html_attributes node.id, node.role
+    case node.type
+    when :strong
+      %(<strong#{attributes}>#{node.text}</strong>)
+    when :emphasis
+      %(<em#{attributes}>#{node.text}</em>)
+    when :monospaced
+      %(<code#{attributes}>#{node.text}</code>)
+    when :mark
+      %(<mark#{attributes}>#{node.text}</mark>)
+    when :superscript
+      %(<sup#{attributes}>#{node.text}</sup>)
+    when :subscript
+      %(<sub#{attributes}>#{node.text}</sub>)
+    when :single
+      attributes = common_html_attributes node.id, node.role, 'single_quote'
+      %(<span#{attributes}>&#8216;#{node.text}&#8217;</span>)
+    when :double
+      attributes = common_html_attributes node.id, node.role, 'double_quote'
+      %(<span#{attributes}>&#8220;#{node.text}&#8221;</span>)
+    else
+      %(<span#{attributes}>#{node.text}</span>)
+    end
+  end
+
+  def convert_inline_break node
+    %(#{node.text}<br />)
```

Comment:
An open question is whether we want to support XHTML or not. And if we want to support XHTML, should we always use `/` on void elements or use an option. Currently, we have an option named `htmlsyntax` to control if we should add `/` or not.

May I ask with the extra space? `<br />` instead of `<br/>` ?

Reply:
Correct me if I'm wrong but XHTML5 is trivially HTML5, no? (maybe except some doctype nonsense). (I've been quite removed from web stuff for a while, so I'm probably rusty on this stuff).

As for `<br />` vs. `<br/>`, I have no opinion, other than I've always put a ` ` personally.

Reply:
It's mostly about adding a slash to close tags. For instance: `<br>` is valid HTML5 but invalid XHTML.